test/runtests.jl

```diff
@@ -0,0 +1,17 @@
+using ChainRules
+using ChainRulesTestUtils
+using Random
+using Test
+
+@testset "ChainRulesTestUtils.jl" begin
+    double(x) = 2x
+    @scalar_rule(double(x), 2)
+    test_scalar(double, 2)
+
+    fst(x, y) = x
+    frule(::typeof(fst), x, y, _, dx, dy) = (x, dx)
+    rrule(::typeof(fst), x, y)=  fst_pullback(dout) = (NO_FIELDS, @thunk(identity(dout)), Zero())
+
+    #frule_test(fst, (x, y))
+    #rrule_test(fst, (x, y))
```

Comment:
This seems to be missing still?
adding this test seems like the last thing left to do.

:woot!